tests/behat/bootstrap/LinksTrait.php

```diff
@@ -27,19 +27,13 @@ public function linkWithTextContains($text, $expectedLink)
         }
     }
 
-    /**
-     * @Given I visit the behat link :link
-     */
     public function visitBehatLink($link)
```

Comment:
I often use these comments to determine the actual step to write? Any reason for removing them?

Reply:
In the case of those specific two, they’re basically private functions so shouldn’t be used in tests directly. Instead we should be calling the test steps which utilise them (e.g. “I open the :regexpr link from the email” ultimately uses `visitBehatLink('get-last-email')`.